a11y/a11y.test.js

```diff
@@ -54,18 +54,30 @@ afterEach(async () => {
 
 test('checks the test page with Axe', async () => {
   jest.setTimeout(600000)
+  // These rules aren't relevant to this project
+  const disabledRules = [
+    'region', // Leaflet does not comply
+    'meta-viewport', // Leaflet does not comply
+    'page-has-heading-one' // Heading is provided by logo
+  ]
+
   // Web security is disabled to allow requests to the mock OTP server
   const browser = await puppeteer.launch({args: ['--disable-web-security']})
-  const page = await browser.newPage()
+  let page = await browser.newPage()
+  // Test trip planner
   await page.goto(`file://${path.resolve(__dirname, '../index-for-puppeteer.html')}#/?ui_activeSearch=0qoydlnut&ui_activeItinerary=0&fromPlace=1900%20Main%20Street%2C%20Houston%2C%20TX%2C%20USA%3A%3A29.750144%2C-95.370998&toPlace=800%20Congress%2C%20Houston%2C%20TX%2C%20USA%3A%3A29.76263%2C-95.362178&date=2021-08-04&time=08%3A14&arriveBy=false&mode=WALK%2CBUS%2CTRAM&showIntermediateStops=true&maxWalkDistance=1207&optimize=QUICK&walkSpeed=1.34&ignoreRealtimeUpdates=true&numItineraries=3&otherThanPreferredRoutesPenalty=900`)
 
-  // These rules aren't relevant to this project
   await expect(page).toPassAxeTests({
-    disabledRules: [
-      'region', // Leaflet does not comply
-      'meta-viewport', // Leaflet does not comply
-      'page-has-heading-one' // Heading is provided by logo
-    ]
+    disabledRules: disabledRules
   })
+  // Test stop viewer
+  page = await browser.newPage()
```

Comment:
I would feel a little better if we weren't reusing this variable. Rec: use two variables:
```javascript
const planTripPage = ...
await planTripPage.goto(....)
const stopViewerPage = ...
await stopViewerPage.goto(....)
```

Reply:
Thank you for refactoring in https://github.com/opentripplanner/otp-react-redux/pull/436